delegate-modules/e2e/checkApplications.cy.ts

```diff
@@ -0,0 +1,65 @@
+import { BaseMethods } from '../../cypress/common/base';
+import { baseSelectors } from '../../cypress/common/selectors';
+import { Constants } from '../../cypress/fixtures/constants';
+
+const basePage: BaseMethods = new BaseMethods()
+
+let appsData = [
+    {
+        headerSelector: baseSelectors.tags.headers.h1,
+        subHeaderSelector: baseSelectors.tags.headers.h2,
+        buttonSelector: baseSelectors.tags.coreElements.button,
+        headerText: Constants.commonConstantsData.basicComponents.basicHostRemote,
+        appNameText: Constants.commonConstantsData.commonCountAppNames.app1,
+        buttonNameText: Constants.updatedConstantsData.commonAppWithButton.app2,
+        host: 3001
+    },
+    {
+        headerSelector: baseSelectors.tags.headers.h1,
+        subHeaderSelector: baseSelectors.tags.headers.h2,
+        buttonSelector: baseSelectors.tags.coreElements.button,
+        headerText: Constants.commonConstantsData.basicComponents.basicHostRemote,
+        appNameText: Constants.commonConstantsData.commonCountAppNames.app2,
+        buttonNameText: Constants.updatedConstantsData.commonAppWithButton.app2,
+        host: 3002
+    }
+]
+
+appsData.forEach(
+    function(
+        property: { 
+            headerSelector: string
+            subHeaderSelector: string
+            buttonSelector: string,
+            headerText: string,
+            appNameText: string,
+            buttonNameText: string,
+            host: number
```

Comment:
You can left there only `appNameText` and `host`. Rest of attr you can use directly in your testcase